android/src/main/java/com/github/wuxudong/rncharts/data/BarDataExtract.java

```diff
@@ -19,6 +20,12 @@
 
 public class BarDataExtract extends DataExtract<BarData, BarEntry> {
 
+    private ReactApplicationContext m_reactContext;
+
+    public BarDataExtract(ReactApplicationContext reactContext) {
+        super();
```

Comment:
Maybe should have the m_reactContext in the DataExtract base class because a bunch of these derive from it